### PR TITLE
IC issue button tells the receiver who hit them with the no u

### PR DESCRIPTION
**Pull request by GuppyLaxx**
Title.

Why?
Why not, it's transparency. I'm sure people get frustrated when they just get hit with the dumbest button that admins have available, so, why not let them know who did it?

:cl: GuppyLaxx
add: IC issue button tells the person which admin did it
/:cl:

### CODE REVIEW

**Comment by MayhemSailor** [comment]
Guppy money when merged. I approve.

File: yogstation/code/modules/admin/verbs/adminhelp.dm
Comment (Amelia0010) [inline yogstation/code/modules/admin/verbs/adminhelp.dm]: check for stealthmin.

**Review by Amelia0010 — CHANGES_REQUESTED** [review]

**Comment by Mr-Space** [comment]
Most admins tell you the reason before hitting it and we don't hit it nilly willy, if it's marked IC, it's IC.

**Review by Amelia0010 — APPROVED** [review]

**Comment by alexkar598** [comment]
Don't merge -BritishGrace over discord reacts

**Comment by GuppyLaxx** [comment]
ok but why not

**Comment by alexkar598** [comment]
Don't merge -SuperGuy over discord reacts

**Comment by Rayisken** [comment]
I suggest lowering the sodium in your diet. It really shouldn't matter who, if you disagree with a decision you can always utilize discord channels or make a complaint on the forums to get other admin input on how your situation was dealt with.

**Comment by GuppyLaxx** [comment]
This is literally about nothing you've just said, please stop delusional, thanks.
How hard is it to comprehend how minuscule of a change this is :
If you're not stealthed, hitting the IC button now tells the person your ckey.
If you're stealthed, it tells them an administrator did it.
That's it.

**Comment by Rayisken** [comment]
"If you're stealthed, it tells them an administrator did it."
They know an admin did it if they pushed the button in the first place.

I don't think insulting people is a productive way of getting this merged. Once again, it really shouldn't matter who pushed the IC button, if you disagree with a decision you can always utilize discord channels or make a complaint on the forums to get other admin input on how your situation was dealt with. If multiple administrators are on, tickets that deal with IC issues are usually discussed in achat prior to being assigned the IC designation making it an all administrator's decision, so it's not always the ticket handler/button pusher that has made the decision.

**Comment by alexkar598** [comment]
Alot of the staff are downvoting this so imma just...

**Comment by Oakboscage** [comment]
I really don't see how this matters either way.

**Comment by GuppyLaxx** [comment]
Ah yes, just close it because staff have the final decision.
if you don't see how it matters, then why not add it?
This is a situation of why not rather than a why, it's literal transparency for those playing, and doesn't affect the admin in any single way

**Comment by ynot01** [comment]
i feel this is a good change

**Comment by Rayisken** [comment]
It's been explained to you very clearly why it does not work as you are expecting

**Comment by GuppyLaxx** [comment]
That literally changes **nothing**. It's amazing to me how it's a literal gang up just because it says it affects admins

**Comment by ThatLing** [comment]
Re-opening this because I asked the HCs what to do with this and alex closed it before they could even respond.

**Comment by alexkar598** [comment]
@GuppyLaxx if it changes nothing then no point having it now

**Comment by alexkar598** [comment]
Time to start marking issues as wiki instead of IC

**Comment by GuppyLaxx** [comment]
It's a small quality of life for normal players. What's wrong with that, even if it's ineffective

**Comment by AsV9** [comment]
God stop being so god damn difficult

**Comment by AffectedArc07** [comment]
Reminder that $100 was paid for this 

why

**Comment by optimumtact** [comment]
Worth it

**Comment by 81Denton** [comment]
Fucking lol

**Comment by Colburn3000** [comment]
Imagine paying someone $100 to add one line of code.

**Comment by MayhemSailor** [comment]
@Colburn3000 imagine $100 being spare change and being worth every penny.

**Pull request merged**
